tests/gates/compress_rotation_approximations_test.py

```diff
@@ -11,8 +11,11 @@ class TestGateCompression:
         [
             ("SSSXHSTHTHSTHTH", ["SSS", "X", "HSTH", "T", "HSTH", "T", "H"]),
             ("SHSSXH", ["S", "H", "SS", "X", "H"]),
+            ("SHSSHX", ["S", "HSSH", "X"]),
             ("HSSTTSHSX", ["HSSTTSH", "S", "X"]),
             ("STXH", ["ST", "X", "H"]),
+            ("HSTX", ["H", "ST", "X"]),
+            ("XHST", ["X", "H", "ST"]),
```

Comment:
We already have a list case, don't you want to add them all?
```
print(partition_gate_sequence('H'))
print(partition_gate_sequence('S'))
print(partition_gate_sequence('HS'))
print(partition_gate_sequence('SH'))
print(partition_gate_sequence('HSH'))
print(partition_gate_sequence('SHS'))
print(partition_gate_sequence('HSHS'))
print(partition_gate_sequence('SHSH'))
print(partition_gate_sequence('HSHSH'))
print(partition_gate_sequence('SHSHS'))
print(partition_gate_sequence('HSHSHS'))
print(partition_gate_sequence('SHSHSH'))
```

Reply:
Yeah I will do that